### PR TITLE
updating Readme.md TOC

### PR DESCRIPTION
**Pull request by HarryMacfinned**
updating the TOC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Portions Copyright (c) 2012-2013 The Cryptonote developers.
   - [Release staging schedule and protocol](#release-staging-schedule-and-protocol)
   - [Compiling Monero from source](#compiling-monero-from-source)
     - [Dependencies](#dependencies)
+  - [Internationalization](#Internationalization)
+  - [Using Tor](#Using Tor)
```
Comment (vtnerd) [inline README.md]: This link is invalid -`#using-tor`. The others (`#internalization` and `#debugging`) are also all lower case, but I _think_ anchor tags are case insensitive. If you mouse-over next to the section name a link icon will appear with the valid anchor names.

Comment (HarryMacfinned) [reply README.md]: Apologies and thanks.
I'll correct in my branch.

**Comment by dEBRUYNE-1** [comment]
@HarryMacfinned - Could you rebase this PR?

**Comment by HarryMacfinned** [comment]
> @HarryMacfinned - Could you rebase this PR?

I have never done that before (I'm not github's best friend right now), but I'll have a look. 
It will probably need some delay.

**Comment by dEBRUYNE-1** [comment]
Thanks. Let us know if you need any help.

**Comment by moneromooo-monero** [comment]
I've rebased it in https://github.com/monero-project/monero/pull/5897/commits

+duplicate

**Pull request closed without merging**